src/packages/application/router/index.spec.tsx

```diff
@@ -45,14 +45,16 @@ describe('RBACLink Component', () => {
 		useLocation.mockReturnValue({ pathname: '/' });
 
 		renderWithAppContext(
-			<Provider store={store}>
-				<RBACLink>
-					<div>Child Component</div>
-				</RBACLink>
-			</Provider>
+			<OidcProvider>
+				<Provider store={store}>
+					<RBACLink>
+						<div>Child Component</div>
+					</RBACLink>
+				</Provider>
+			</OidcProvider>
 		);
 
-		expect(screen.getByText('Child Component')).not.toBeNull();
```

Comment:
ça ne fonctionne pas :
`TestingLibraryElementError: Unable to find an accessible element with the role "div" and name "Child Component"`